clients/naming_client/naming_cache/service_info_holder_test.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 1999-2020 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package naming_cache
+
+import (
+    "testing"
+)
+
+func TestNewServiceInfoHolder(t *testing.T) {
+    NewServiceInfoHolder("", "./cache", true, true)
```

Comment:
Unit tests should have assertions

Reply:
I had remove the test case, this seems unnecessary